src/GitTools.jl

```diff
@@ -85,6 +85,10 @@ function setprotocol!(;
 end
 
 function normalize_url(url::AbstractString)
+    # libgit2 is fussy about trailing slash. Make sure there is none.
+    while endswith(url, "/")
```

Comment:
`rsplit(url, "/")[1]`?

Reply:
```
julia> a = "fooα/"
"fooα/"

julia> a[1:end-1]
ERROR: StringIndexError("fooα/", 5)
```

Reply:
`strip(url, '/')` removes them all

Reply:
I would suggest replacing all of this with `url = rstrip(url, '/')`

Reply:
Finally we have a winner.